content/cpp/concepts/vectors/terms/size/Capacity/Capacity.md

```diff
@@ -0,0 +1,59 @@
+---
+Title: '.capacity()'
+Description: 'Returns the storage space currently allocated for the vector'
+Subjects:
+  - 'Computer Science'
+  - 'Game Development'
+Tags:
+  - 'Vectors'
+  - 'Arrays'
+  - 'Data Structures'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **.capacity()** method returns the storage space allocated for a vector. This is different from .size(). Capacity can be equal or greater than size. It is the total number of elements that a vector can hold without reallocation. This is automatically assigned at initialization.
```

Comment:
```suggestion
The **`.capacity()`** method returns the storage space currently allocated for a vector. This differs from [`.size()`](https://www.codecademy.com/resources/docs/cpp/vectors/size), which represents the number of elements currently in the vector. Capacity can be equal to or greater than size. It indicates the total number of elements the vector can hold without requiring reallocation. This capacity is automatically assigned when the vector is initialized.
```

Reply:
Modifications:
1. the first instance of the term should be in **bold** and `backticks`
2. Reframed it for better clarity
3. added the description and link for the `.size()` method

Reply:
Hi,
I have a suggestion. Isn't it better to write `This value is automatically assigned when the vector is initialized` instead of `This capacity is automatically assigned when the vector is initialized`? Since I am defining `.capacity()`.